pipelines/internal/commands/run/run.go

```diff
@@ -171,6 +171,7 @@ var (
 	sharePIDs      = flags.Bool("share-pids", false, "if true, all actions will share the same PID namespace")
 	cosChannel     = flags.String("cos-channel", "", "if set, specifies the COS release channel to use")
 	serviceAccount = flags.String("service-account", "", "if set, specifies the service account for the VM")
+	logsInterval   = flags.Duration("logsInterval", 0, "if non-zero, specifies the time interval for logging output during runs")
```

Comment:
Follow flags style and use the same name to correlate with the output flag.

--output-interval

(The reason it's called output and not logs is because it's only one piece of the logs that are generated - the combined process output.)

Reply:
Done.